scip/api/population.py

```diff
@@ -0,0 +1,91 @@
+from salmon_occurrence import ConservationUnit, Taxon, Reference, Population, Phenology
+from sqlalchemy_sqlschema import maintain_schema
+from sqlalchemy_sqlschema.sql import get_schema
+from sqlalchemy import func
+
+
+def population(
+    session,
+    overlap=None,
+    common_name=None,
+    scientific_name=None,
+    subgroup=None,
+    name=None,
+):
+    """Return information about salmon populations in the database that fulfill
+    specified parameters.
```

Comment:
My reading the code, my understanding that it returns a logical AND of all provided conditions and that none of the parameters are required. If that's the case, it would be worth making that explicit in the docstring.